### ISSUE TITLE
Suggestion to include better description and package method documentation

### ISSUE THREAD

**Issue by bigdoods** [googleapis/nodejs-cloud-container#145]
Hi,

I am new to using the Kubernetes API and GKE and I just spent a considerable amount of time this morning looking at this package to create a deployment. It was not until I read the documentation [here](https://cloud.google.com/kubernetes-engine/docs/reference/api-organization) on the API organisation and structure until I realised that I need to use the Kubertnetes API and a node-client to make deployments. It would be a good idea to warn people that they can only do cluster management with this package and perhaps change the description of the repo to something more descriptive than "Deploy, manage, and scale containerized applications on Kubernetes, powered by Google Cloud." perhaps removing the _Deploy_ word altogether.

I also had to dig into the code to view all of the methods that were exposed for GKE engine API. It would be nice to have these documented in the Readme or somewhere more front and foremost.

Good luck!

**Comment by bcoe** [comment]
addressed by https://github.com/googleapis/nodejs-cloud-container/pull/207 once it lands.
